runtime/src/bank.rs

```diff
@@ -2210,35 +2218,35 @@ mod tests {
             keypairs[8].pubkey(),
             Account::new(
                 generic_rent_due_for_system_account + 2 + 929,
-                1,
+                0,
                 &Pubkey::default(),
             ),
         ));
         account_pairs.push((
             keypairs[9].pubkey(),
-            Account::new(10, 1, &Pubkey::default()),
+            Account::new(10, 0, &Pubkey::default()),
         ));
 
         // Feeding to MockProgram to test read only rent behaviour
         account_pairs.push((
             keypairs[10].pubkey(),
             Account::new(
                 generic_rent_due_for_system_account + 3,
-                1,
+                0,
```

Comment:
@t-nelson if there's any code that would barf on a system account with data that isn't a Nonce, might be a good idea to see what things it's trying to do.  it *is* possible to create a system account with data, now, wouldn't want that to be a lamport black-hole

Reply:
The tests I checked that actually used the data were just forcing some rent collection.  A few others seemed to be c/p from the rent tests and didn't use the data at all.  I'll give them all a more thorough going over in any case.

Reply:
More thorough examination of the tests that broke confirms the above